### PR TITLE
ignoring some useless things

### PR DESCRIPTION
**Pull request by LucasSantiag**

### CODE REVIEW

File: src/compiler/scala/tools/nsc/classpath/AggregateClassPath.scala
Comment (som-snytt) [inline src/compiler/scala/tools/nsc/classpath/AggregateClassPath.scala:32]: FWIW, unused warning is suppressed by `simpleClassName @ _` which is lexically equivalent but visually signals that the name is introduced only to document what value is unused. Probably none of these examples are obscure; here, the RHS says "pkg and class name". Could also `rhs._1`.

Comment (LucasSantiag) [reply src/compiler/scala/tools/nsc/classpath/AggregateClassPath.scala:32]: Do you think this would be better?
```
val pkg = PackageNameUtils.separatePkgAndClassNames(className)._1
```

File: src/compiler/scala/tools/nsc/classpath/AggregateClassPath.scala
Comment (diesalbla) [inline src/compiler/scala/tools/nsc/classpath/AggregateClassPath.scala]: A matcher may be slightly more readable. The tuple is probably optimised away.
```suggestion
    val (pkg, _) = PackageNameUtils.separatePkgAndClassNames(className)
```

File: src/compiler/scala/tools/nsc/classpath/AggregateClassPath.scala
Comment (diesalbla) [inline src/compiler/scala/tools/nsc/classpath/AggregateClassPath.scala]: Same here.

**Review by diesalbla — APPROVED** [review]

File: src/compiler/scala/tools/nsc/classpath/AggregateClassPath.scala
Comment (som-snytt) [reply src/compiler/scala/tools/nsc/classpath/AggregateClassPath.scala]: I don't have strong opinion on what is most readable. I see all the styles in this codebase. I'll defer to anyone who cares to either approve or object.

Comment (LucasSantiag) [reply src/compiler/scala/tools/nsc/classpath/AggregateClassPath.scala]: Yeah, I agree with you

**Comment by LucasSantiag** [comment]
@diesalbla alterations done!

**Comment by SethTisue** [comment]
thank you, and welcome to the repo!

**Comment by SethTisue** [comment]
(oh, oops, I should have asked for this to be squashed before merge. nbd)

**Pull request merged**
